index.html

```diff
@@ -400,6 +400,21 @@ <h3>Timeline</h3>
 
 			<section id="success-criteria">
 				<h2>Success Criteria</h2>
+				
+				<p>The EPUB 3 Working Group’s work is considered a success 
+				if all of the following conditions are met:</p>
+				
+				<ol>
+					<li>There are multiple, independent, interoperable implementations 
```

Comment:
I'm worried this statement might be baking in a lot fudge. What is independent? Would two skins of readium count as independent implementations? How is interoperability gauged? Does it mean anything more than two implementations? What constitutes "widely used"? Is within only one country okay? Does the size of the ebook market being served matter? I can see it getting picked on in these ways.

Reply:
These are the standard terms in the charters and it is deliberately a big vague. And that it intentional: these answers will depend on the nature of the nature of the group.. It is something that the WG will have to define when the time comes. At the end of the day, the Director has to accept the CR criteria as proposed by the WG and the W3C membership, via the AC, has the right to object to the criteria when the CR is announced.

Reply:
The larger point is that if we have features of EPUB with only two implementations, I would think of that as a failure. Given the maturity of EPUB, and the huge number of existing content and implementations, I think the bar must be much higher than for brand-new specifications.

Reply:
> The larger point is that if we have features of EPUB with only two implementations, I would think of that as a failure.

Speaking pessimistically, support tends to trickle in later as budgets are approved and support/release timelines figured out. I'll be pleasantly surprised to see the opposite be true for once, but it would be more of a blocker to rec if indeed these are formal requirements. (A lot depends on what we consider a feature of the standard, too.)

Would allowing for verbal commitments making things better or worse? In particular, something like: "There is stated commitment from key vendors (i.e., with extensive global or regional reach) to support the standard, either by the time it becomes a recommendation or in a timely manner thereafter." That's sort of the ideal we're after, right?

Reply:
I'm also thinking of how things work in other parts of W3C. I don't think there's any CSS module that's gotten to REC without being in every browser. The CSSWG is perhaps too comfortable with leaving things in CR for years and years, but the RECs are very solid.

Reply:
The charter sets the minimal requirement from the process point of view. The WG has the obligation to submit a final set of CR exit criteria when the time comes; those can be more stringent than what the charter says. But, at this moment, we are talking about the charter.

B.t.w., looking at the CSS case: the [CSS WG Charter](https://www.w3.org/2019/10/css-wg-charter.html) uses essentially the same minimal wording. The fact that the WG requires all browsers to implement the features is an extra obligation that the WG has taken on. Which is in line with what I said...

Reply:
> B.t.w., looking at the CSS case: the CSS WG Charter uses essentially the same minimal wording.

It sounds more aspirational, though, which is maybe the worry I had with our statement. There's something about "the following conditions are met" that suggests a requirement.

End of the day, it's not a major issue to me, but a last suggestion would be to strip off that part:

"The EPUB 3 Working Group’s work is considered a success if:
- It results in multiple, ..."

The timing and specifics of the implementations are then a little more open-ended, whatever more specific exit criteria we might devise later.

Reply:
Fixed via 7c59b1a393c49c1003e319bf1b2fffcdacbfd522